File: barikoitrace/src/main/java/com/barikoi/barikoitrace/p000b/p002d/LocationUpdateListener.java
package com.barikoi.barikoitrace.p000b.p002d;

import android.location.Location;

import com.barikoi.barikoitrace.models.BarikoiTraceError;


public interface LocationUpdateListener {

    void onLocationReceived(Location location);

    void onFailure(BarikoiTraceError barikoiError);
}

File: barikoitrace/src/main/java/com/barikoi/barikoitrace/location/LocationUpdateListener.kt
package com.barikoi.barikoitrace.location

import android.location.Location
import com.barikoi.barikoitrace.model.TraceError

interface LocationUpdateListener {
    fun onLocationReceived(location: Location)
    fun onFailure(error: TraceError)
    fun onProviderAvailabilityChanged(available: Boolean)
}
